server/welcomebot.go

```diff
@@ -135,7 +135,7 @@ func (p *Plugin) renderWelcomeMessage(messageTemplate MessageTemplate, configMes
 
 	tmpMsg, _ := template.New("Response").Parse(strings.Join(configMessage.Message, "\n"))
 	var message bytes.Buffer
-	tmpMsg.Execute(&message, messageTemplate)
+	_ = tmpMsg.Execute(&message, messageTemplate)
```

Comment:
Logging these errors is valuable, as it can greatly help with debugging

Reply:
I have added a simple log of this error in [`685000`](https://github.com/mattermost/mattermost-plugin-welcomebot/pull/51/commits/685000ca2f2eda5812ce7ef2b09451a69774816e). Is this sufficient ?